odb_extractor_atoms.py

```diff
@@ -160,7 +160,9 @@ def findatoms(observation):
             atoms[-1]['part_time'] += step_time
         else:
             atoms[-1]['prog_time'] += step_time
-
+            
+#         atoms[-1]['id'] = atomlabel
+            
         print('{:22} {:12} {:7.2f} {:3d} {:10} {:15} {:12} {:12} {:7.4f} {:5.2f} {:5.2f} {:3d}'.format(datalab,
```

Comment:
Again, just a nitpick, so we don't have to do this right away, but for the future, let's keep away from `.format` and use `f`-strings instead, as this is the best practice moving forward from what I can see.